@c-nuro can you please fix the static checks?
```
airflow/providers/google/cloud/transfers/gcs_to_bigquery.py:23:1: F401 'airflow.exceptions.AirflowException' imported but unused
```

--- a/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py
+++ b/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py
@@ -20,7 +20,6 @@
 import json
 from typing import Optional, Sequence, Union
 
-from airflow.exceptions import AirflowException
 from airflow.models import BaseOperator
 from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook
 from airflow.providers.google.cloud.hooks.gcs import GCSHook

